WordPress/Classes/Models/PostType.h

```diff
@@ -0,0 +1,16 @@
+#import <CoreData/CoreData.h>
+
+@class Blog;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface PostType : NSManagedObject
+
+@property (nullable, nonatomic, retain) NSString *label;
+@property (nullable, nonatomic, retain) NSString *name;
+@property (nullable, nonatomic, retain) NSNumber *apiQueryable;
+@property (nullable, nonatomic, retain) Blog *blog;
```

Comment:
For this file, this is the generated code from CoreData when generating via "Create NSManagedObject subclass". Seems wonky, I know, but I tend to lean on whatever CoreData generates. We could change it though.